src/packages/math/src/euler.js

```diff
@@ -1,8 +1,8 @@
 import MathArray from './math-array';
 import {checkNumber, clamp} from './common';
-import {Matrix4} from './matrix4';
-import {Quaternion} from './quaternion';
-import {Vector3} from './vector3';
+import Matrix4 from './matrix4';
+import Quaternion from './quaternion';
+import Vector3 from './vector3';
```

Comment:
> But should be on a separate PR?

Yes, I suppose that could have made sense. This PR could have been broken into three really small pieces. I'll be a little less trigger happy with the merge button going forward.